infra/terraform/modules/data_access/organization_events/tests/test_organization_events.py

```diff
@@ -4,32 +4,26 @@
 
 import organization_events
 
-
-@pytest.mark.usefixtures(
-    "given_the_env_is_set",
-    "given_lambda_is_available",
-)
-def test_event_received_member_added(lambda_client_mock, member_added_event, create_role_arn, payload_bytes):
-    organization_events.event_received(member_added_event, None)
-
-    # Test right lambda function is invoked asyncronously
-    lambda_client_mock.invoke.assert_called_with(
-        FunctionName=create_role_arn,
-        Payload=payload_bytes,
-        InvocationType="Event",
-    )
+from tests.conftest import PAYLOAD_BYTES
+from tests.conftest import sns_event
+from tests.conftest import TEST_CREATE_ROLE_ARN
+from tests.conftest import TEST_DELETE_ROLE_ARN
 
 
+@pytest.mark.parametrize("event,lambda_arn", [
```

Comment:
This is how I ended up handling the `pytest.mark.parametrize`, I tweaked the `sns_event()` function to just get the GH event name and I'm using it here. It seems nicer but have a look, you may have some feedback.